app/assets/scripts/resources.js

```diff
@@ -0,0 +1,13 @@
+$(document).ready(function () {
+    // This is for search on the index page
+    $('#search-resources').submit(function(e) {
+        e.preventDefault();
+        var query = document.getElementById('search-resources-text').value;
+        console.log(query);
+        var endpoint = '/single-resource/' + query;
+        if (query.length == 0) {
+            endpoint = '/single-resource/';
```

Comment:
if query is an empty string wouldn't the endpoint be '/single-resource/' from line 7?